src/_locales/en_US/messages.json

```diff
@@ -91,6 +91,10 @@
         "message": "Prevent WebRTC from leaking local IP address",
         "description": "Checkbox label on the general settings page"
     },
+    "options_allow_sites_widget_replacement": {
+        "message": "Always allow widgets on these sites:",
+        "description": "Text above the options page area that lets users add or remove domains from the widget site allow list."
```

Comment:
We don't allow removal anymore ... something succinct like, "Label for the widget site exceptions form"?